todo_app/data/trello_items.py

```diff
@@ -10,7 +10,7 @@ def add_item(title):
     query_params = {
         "key": os.getenv("TRELLO_API_KEY"),
         "token": os.getenv("TRELLO_API_TOKEN"),
-        "idList": os.getenv("TODO_LIST_ID"),
+        "idList": os.getenv("TRELLO_TODO_LIST_ID"),
```

Comment:
Don't forget to update your `.env.template` file to warn future developers that this has changed!